modules/tcia_dataset.ipynb

```diff
@@ -1,728 +1,863 @@
 {
```

Comment:
<p>thanks Justin, please clear this output, and the rest of the notebook updates look great.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>